modules/xfeatures2d/samples/shape_transformation.cpp

```diff
@@ -2,10 +2,10 @@
  * shape_context.cpp -- Shape context demo for shape matching
  */
 #include <iostream>
+#include "opencv2/shape.hpp"
```

Comment:
@sovrasov we need to determine `HAVE_OPENCV_SHAPE` is defined or not. in actual code `HAVE_OPENCV_SHAPE` is always undefined. by this change it works as it should be